packages/web3-eth/test/integration/web3_eth/send_signed_transaction.test.ts

```diff
@@ -59,15 +65,25 @@ describe('Web3Eth.sendSignedTransaction', () => {
 				...transaction,
 				...gasPricing,
 			});
-			const response = await web3Eth.sendSignedTransaction(signedTransaction.raw);
-			expect(response.status).toBe('0x1');
+			const response = await web3Eth.sendSignedTransaction(
+				typeof signedTransaction.raw === 'string'
+					? signedTransaction.raw
+					: signedTransaction.raw.raw,
```

Comment:
There shouldn't be nested raw for output of `web3Eth.signTransaction`. I think it has to do with recently discussed geth vs ganache output diff?

Reply:
Yes this change didn't appear from this PR. It was done earlier in @spacesailor24 PR and relates to the output difference for ganache and geth.

Reply:
ok, https://github.com/ChainSafe/web3.js/issues/5077 with this issue we will need to fix these tests as well.